stdlib/kokorawr/kokorawr/functions/services/bs.js

```diff
@@ -188,21 +185,14 @@ module.exports = (team=0, x='C', y=5, reset=false, turn=false, state=false, cont
     } else if(reset) {
       return callback(null, {text: "Can't reset: game not over\n" + boardToString(boards[team]), success: false})
     } else if(lastTeam == team) {
-      let color = 'Red';
-        if(team != 0) {
-          color = 'Blue';
-        }
-      return callback(null, {text: `It is not ${color}'s turn.`, success: false})
+      return callback(null, {text: `It is not your turn. It is ${color}'s turn!`, success: false})
     } else if (boards[team][xInd][y] != EMPTY){
```

Comment:
For anything with a string comparison, make it === or !==

Reply:
nevermind seems fine on closer inspection

Reply:
🎉